element_miniscope/miniscope.py

```diff
@@ -75,20 +64,12 @@ def activate(
 
 
 def get_miniscope_root_data_dir() -> list:
-    """
-    All data paths, directories in DataJoint Elements are recommended to be stored as
-    relative paths (posix format), with respect to some user-configured "root"
-    directory, which varies from machine to machine (e.g. different mounted drive
-    locations).
-
-    get_miniscope_root_data_dir() -> list
-        This user-provided function retrieves the possible root data directories
-         containing the miniscope data for all subjects and sessions
-         (e.g. acquired Miniscope-DAQ-V4 raw files,
-         output files from processing routines, etc.).
-
-        :return: a string for full path to the root data directory,
-         or list of strings for possible root data directories
+    """Fetches absolute data path to miniscope data dir(s).
+    
+    The absolute path here is used as a reference for all downstream relative paths used in DataJoint.
```

Comment:
Does this element/workflow accept multiple root dirs? Unless we're sure that's the case, let's change to all references here to singular